### PR TITLE
Add support for linux brew @latest

### PR DESCRIPTION
**Pull request by geemanjs**

### CODE REVIEW

File: Abstract/abstract-drone.rb [hunk 1]
```diff
@@ -4,14 +4,22 @@ class AbstractDrone < Formula
...
-    url assets.select { |v| File.basename(v['browser_download_url']) == 'drone_darwin_amd64.tar.gz' }.first['browser_download_url']
+    url assets.select { |v| File.basename(v['browser_download_url']) == "#{platform_url}" }.first['browser_download_url']
     sha256 sha256sum
 
     test do
       system "#{bin}/drone", "--version"
     end
   end
 
+  def self.platform_url
```
Comment (tboerger) [inline Abstract/abstract-drone.rb]: It's not an url, beside that maybe directly detect everything automatically? Like the platform and the architecture?

**Comment by geemanjs** [comment]
@tboerger Good point RE platform/architecture! 

Not sure my ruby skills are up to figuring that out - this is the first ruby I've ever written :+1: (and unfortunately don't have the time right now to learn more).

I've corrected the `platform_url`.

If I do some more ruby I'll be sure to come back and add that. This suffices for my projects current needs so happy for you to accept/reject.

File: Abstract/abstract-drone.rb [hunk 2]
```diff
@@ -4,14 +4,22 @@ class AbstractDrone < Formula
   def self.init
     homepage "https://github.com/drone/drone-cli"
     head "https://github.com/drone/drone-cli.git"
-    url assets.select { |v| File.basename(v['browser_download_url']) == 'drone_darwin_amd64.tar.gz' }.first['browser_download_url']
+    url assets.select { |v| File.basename(v['browser_download_url']) == "#{drone_file_name}" }.first['browser_download_url']
```
Comment (tboerger) [inline Abstract/abstract-drone.rb:7]: Just `drone_file_name` instead of `"#{drone_file_name}"`

File: Abstract/abstract-drone.rb [hunk 3]
```diff
@@ -4,14 +4,22 @@ class AbstractDrone < Formula
...
     sha256 sha256sum
 
     test do
       system "#{bin}/drone", "--version"
     end
   end
 
+  def self.drone_file_name
+    p = 'drone_darwin_amd64.tar.gz'
```
Comment (tboerger) [inline Abstract/abstract-drone.rb:16]: Maybe more like that:

```ruby
def self.drone_file_name
  if RbConfig::CONFIG['host_os'].include? 'darwin'
    'drone_darwin_amd64.tar.gz'
  else
    'drone_linux_amd64.tar.gz'
  end
end
```

**Pull request merged**